src/main/java/com/unosquare/cvgenerator/model/entity/Greeting.java

```diff
@@ -3,13 +3,15 @@
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
 
 @Entity
 @Table(name = "hello")
 @Data
 @AllArgsConstructor
+@NoArgsConstructor
 @Builder
```

Comment:
Suggestion - use only one of `@AllArgsConstructor` or `@Builder`, shouldn't need both